src/lib/formatter/SarifFormatter.ts

```diff
@@ -6,6 +6,7 @@ import { RuleCatalog } from '../services/RuleCatalog';
 import { Controller } from '../../Controller';
 import { ESLint } from 'eslint';
 import * as retire from 'retire';
+import {Location, Log, Notification, Region, Run, ReportingDescriptor} from 'sarif';
```

Comment:
These types let us get rid of most of the `unknown`s and `any`s that were floating around this class.